### DISCUSSION TITLE
i want to run the installation in the background

### DISCUSSION THREAD

**Discussion by mimikun** [nvim-treesitter/nvim-treesitter#6220] [Q&A]
I have and want a lot of parsers installed.
When setting up for the first time, you must press and hold the Enter key or press it repeatedly.
This is very troublesome.
I want it to run in the background like mason.

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/13450321/5c9f80c8-6bdc-4f3d-b6af-c5057ed05f23)

**Comment by clason** [comment]
Please try the `main` branch (which will become the default 1.0). The `master` branch is soft-frozen, and we will not make any more changes there.
